samples/boards/stm32/power_mgmt/blinky/boards/stm32l562e_dk.conf

```diff
@@ -0,0 +1,4 @@
+# Disable DEBUG in order to enter core low power states.
+# WARNING: requires to erase the device (using STM32CubeProgrammer for instance)
```

Comment:
may be useful to let user know how this can be done using the `stm32cubeprogrammer` runner

Reply:
Well, on this point, I was seeing issues that I had no time to dig.
When running `west flash -r stm32cubeprogrammer -d build/stm32l562e_dk --erase`, I can still see the led blinking, even using the same option than with the UI.
But that could likely be a stm32cubeprogrammer issue.

Reply:
@gmarull I've made few more tests and I can't find an handy way to use stm32cubeprogrammer for now.
Anyway, I think it would be wiser sticking with doc changes at this step if that's fine with you